documentation/operators/src/archive/nodes/mix-node-setup.md

```diff
@@ -1,23 +1,27 @@
 # Mix Nodes
 
-> The Nym Mix Node binary was built in the [building nym](../binaries/building-nym.md) section. If you haven't yet built Nym and want to run the code, go there first.
+```admonish warning
+**This is an archived page for backwards compatibility. The content of this page is not updated since April 19th 2024. Eventually this page will be terminated!**
```

Comment:
See note above on `admonish warning` in gateway archive

Reply:
done